src/main/java/com/lineate/testyourlexicon/dto/AnswerResponseDto.java

```diff
@@ -0,0 +1,18 @@
+package com.lineate.testyourlexicon.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class AnswerResponseDto {
+
+  @JsonProperty("guessed")
+  private boolean guessed;
+  @JsonProperty("correct_answer")
```

Comment:
Usually jsons are camelCased

Reply:
All my jsons are snake cased, should I change all of them?